erts/doc/src/erlang.xml

```diff
@@ -12127,10 +12131,15 @@ improper_end</pre>
         <pre>
 > <input>trunc(36028797018963969.0).</input>
 36028797018963968</pre>
-<p>In the last example, <c>trunc(36028797018963969.0)</c> evaluates to
-<c>36028797018963968</c>. The reason for this is that <seeguide
-marker="system/reference_manual:data_types#float_representation_problem"><c>36028797018963969.0</c>
-cannot be represented exactly as a float value</seeguide>.</p>
+      <p>In the last example, <c>trunc(36028797018963969.0)</c>
+      evaluates to <c>36028797018963968</c>. The reason for this is
+      that the float literal <c>36028797018963969.0</c> gets converted
+      to the float value <c>36028797018963968.0</c> as
+      <c>36028797018963968.0</c> is the closest value that can be
+      represented exactly as a float value. See <seeguide
```

Comment:
I think this sentence can be improved. Repetitions of 36028797018963968.0 and the word "value" and the formulation "gets converted" seems odd.

How about something like this?: "The reason for this is that the number 36028797018963969.0 cannot be represented exactly as a float value. Instead, the float literal is represented as 36028797018963968.0, which the closest number that can be represented exactly as a float value."

Reply:
Thanks, I like your suggestion. I will add a commit that changes the text according to your suggestion.

Reply:
Fixed in 9443da9